jbt-it/neomdb
# #205 update jwt after permission change

Added jwt update after every refresh of the website.

How to test:
- Log in with w.luft and navigate to a member profile
- Open an(other) incognito window and log in with m.decker and add (or remove) the member management permission to (from) the w.luft (you can also use the swagger /api/docs to do this)
- Go to the other window with w.luft and refresh the page
- Now w.luft should be able to edit the profile (or is not able anymore depending on if you removed or added the permission)
